src/patternfly/components/DataList/data-list.scss

```diff
@@ -67,6 +67,14 @@
   --pf-c-data-list__cell--cell--PaddingTop: 0;
   --pf-c-data-list__cell--cell--md--PaddingTop: var(--pf-global--spacer--lg);
   --pf-c-data-list__cell--m-icon--cell--PaddingTop: var(--pf-global--spacer--lg);
+  --pf-c-data-list__cell--MinWidth: initial;
+  --pf-c-data-list__cell--MaxWidth: 100%;
+  --pf-c-data-list__cell--Overflow: visible;
+  --pf-c-data-list__cell--TextOverflow: clip;
+  --pf-c-data-list__cell--WhiteSpace: normal;
+  --pf-c-data-list__cell--WordBreak: normal;
+  --pf-c-data-list__cell--m-truncate--MaxWidth: 100%;
```

Comment:
```suggestion
  --pf-c-data-list--cell--MinWidth: initial;
  --pf-c-data-list--cell--MaxWidth: 100%;
  --pf-c-data-list--cell--Overflow: visible;
  --pf-c-data-list--cell--TextOverflow: clip;
  --pf-c-data-list--cell--WhiteSpace: normal;
  --pf-c-data-list--cell--WordBreak: normal;
  --pf-c-data-list--cell--m-truncate--MaxWidth: 100%;
```

Reply:
I'm not too sure about this - since its being applied to `pf-c-data-list__cell` so sholdn't it match that ? there are also all these vars for compact with `--pf-c-data-list--m-compact__cell--PaddingTop`